src/js/components/DataFilter/DataFilter.js

```diff
@@ -118,11 +118,17 @@ export const DataFilter = ({
 
   const id = `${dataId}-${property}`;
 
+  // only add aria-label for no form examples
+  const ariaLabel = noForm
```

Comment:
open to naming suggestion if we wanted to change here

Reply:
This variable name is fine to me